boilerplate/flyte/end2end/Makefile

```diff
@@ -0,0 +1,9 @@
+# WARNING: THIS FILE IS MANAGED IN THE 'BOILERPLATE' REPO AND COPIED TO OTHER REPOSITORIES.
+# ONLY EDIT THIS FILE FROM WITHIN THE 'FLYTEORG/BOILERPLATE' REPOSITORY:
+#
+# TO OPT OUT OF UPDATES, SEE https://github.com/flyteorg/boilerplate/blob/master/Readme.rst
+
+.PHONY: end2end_execute
+end2end_execute:
+	chmod a+x ./boilerplate/flyte/end2end/end2end.sh
```

Comment:
I added `chmod` for solving the permission issue but I am not sure why it started after migration

Reply:
You can do this locally... and then add the file to git... I believe this is one of the things that get persisted in git... does that make sense?

``
chmod a+x ....
git add -A
git commit...
``

Reply:
Do you still need this?

Reply:
No